content/v1.11/getting-started/provider-aws-part-2.md

```diff
@@ -87,7 +87,7 @@ EOF
 
 ## Create a composition
 [Part 1]({{<ref "provider-aws" >}}) created a single _managed resource_.
-A _Composition_ is a template to create multiple _managed resources_ at the same time.
+A _Composition_ is a template to create more than one _managed resource_ at the same time.
```

Comment:
I believe this would be more correct:
```suggestion
A _Composition_ is a template to create one or more _managed resources_ at the same time.
```

Reply:
> I believe this would be more correct:

I agree. There are a few more instances of this that need to be corrected as well.

Reply:
Fixed with other references changed.